src/engraving/rw/inoutdata.h

```diff
@@ -47,6 +47,7 @@ struct ReadInOutData {
     // for except - in
     ReadLinks links;
     std::optional<double> overriddenSpatium = std::nullopt;
+    std::set<Pid> stylesToSkip;
```

Comment:
Something is wrong here :) It either should use mu::engraving::Sid or should be renamed to propertiesToSkip